internal/rpcproxy/rpcproxy.go

```diff
@@ -735,6 +740,20 @@ func (p *clientProxy) handleAdminFacade(ctx context.Context, msg *message) (clie
 
 		return controllerLoginMessageFnc(user)
 	case "Login":
+		var request params.LoginRequest
```

Comment:
I'd prefer this in a func called "handleAnonymousLogin" (whole lot can go in because we'll never support basic logic anyway), and then on that func godoc it so it's clear why we're supporting it and what it aims to achieve.

Reply:
Changed.